<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoPanel</class>
 <widget class="QGroupBox" name="UserInfoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Skin</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QOpenGLWidget" name="openGLWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>252</width>
          <height>306</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Scoreboard Objectives</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="scoreboard_table">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="cornerButtonEnabled">
               <bool>false</bool>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Value</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="kick_user">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Kick</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ban_user">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ban</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>scoreboard_table</sender>
   <signal>itemEntered(QTableWidgetItem*)</signal>
   <receiver>UserInfoPanel</receiver>
   <slot>ScoreboardEnter(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scoreboard_table</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>UserInfoPanel</receiver>
   <slot>ScoreboardValue(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kick_user</sender>
   <signal>clicked()</signal>
   <receiver>UserInfoPanel</receiver>
   <slot>UserKick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ban_user</sender>
   <signal>clicked()</signal>
   <receiver>UserInfoPanel</receiver>
   <slot>UserBan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetUser(QString)</slot>
  <slot>Update()</slot>
  <slot>ScoreboardValue(QTableWidgetItem*)</slot>
  <slot>ScoreboardEnter(QTableWidgetItem*)</slot>
  <slot>UserKick()</slot>
  <slot>UserBan()</slot>
 </slots>
</ui>
